ui/app/AppLayouts/Communities/stores/CommunitiesStore.qml

```diff
@@ -251,5 +248,9 @@ QtObject {
         function onCommunityInfoAlreadyRequested() {
           root.communityInfoAlreadyRequested()
         }
+
+        function onCommunityInfoRequestCompleted(communittyId, erorrMsg) {
+            root.communityInfoRequestCompleted(communittyId, erorrMsg)
```

Comment:
```suggestion
        function onCommunityInfoRequestCompleted(communityId, erorrMsg) {
            root.communityInfoRequestCompleted(communityId, erorrMsg)
```

Reply:
You might want to fix the `erorrMsg` typo too :laughing:

Reply:
Damn, the community typo was so egregious I missed the second typo :joy: